### DISCUSSION TITLE
升级umi/max的plug里的依赖项，是直接本地安装新版本吗

### DISCUSSION THREAD

**Discussion by WuJiY** [umijs/umi#11774] [Q&A] [answered]
比如我想升级max里的react-query到最新版5.0，是直接在本地安装对应的5.0就可以了吗？还是必须要等官方umi/max里升级？

**Comment by fz6m** [comment, marked as answer]
RQ v4 和 v5 有不少 Breaking change 不同的写法，正常情况很难升级，现在 v4 就可以一直使用，无需升级。

如果是新项目或者较小规模有能力升级的项目，可以用 v5 ，这种情况不用开启 max 的 `reactQuery: {}` ，自己安装 v5 版本的 RQ 之后自己手动配置 provider 使用即可。如有兴趣欢迎 PR 兼容 v5 版本的 RQ 。
